Docker/README.md

```diff
@@ -1,29 +1,64 @@
+    - [Objectives of the Docker Lab](#objectives-of-the-docker-lab)
+    - [Reference documents](#reference-documents)
+    - [Note on Linux commands](#note-on-linux-commands)
+- [Environment setup](#environment-setup)
+    - [Proxy consideration](#proxy-consideration)
+    - [Docker installation](#docker-installation)
+        - [Ubuntu installation](#ubuntu-installation)
+        - [RHEL installation](#rhel-installation)
+        - [Check installation](#check-installation)
+- [Using Docker](#using-docker)
+    - [The first container](#the-first-container)
+    - [The second container](#the-second-container)
+- [Configuring owncloud in a container](#configuring-owncloud-in-a-container)
+- [Using Docker compose](#using-docker-compose)
+    - [Installing Docker compose](#installing-docker-compose)
+    - [Our first Docker-compose.yml file](#our-first-docker-composeyml-file)
+    - [Going further with Docker-compose.yml](#going-further-with-docker-composeyml)
+- [Using Docker-machine to create Docker hosts](#using-docker-machine-to-create-docker-hosts)
+- [Using Docker Swarm](#using-docker-swarm)
+    - [Installing Docker Swarm](#installing-docker-swarm)
+    - [Installing on RHEL 7](#installing-on-rhel-7)
+    - [Installing on Ubuntu](#installing-on-ubuntu)
+    - [Installing the engine manually](#installing-the-engine-manually)
+    - [Installing the engine in the Cloud](#installing-the-engine-in-the-cloud)
+    - [Using Docker Swarm to make our configuration available and scalable](#using-docker-swarm-to-make-our-configuration-available-and-scalable)
+        - [CentOS/RHEL](#centosrhel)
+        - [Ubuntu](#ubuntu)
+- [Deploy a cloud native application.](#deploy-a-cloud-native-application)
+    - [Objectives](#objectives)
+
+<!-- /TOC -->
 
 ## Objectives of the Docker Lab
-At the end of the Lab students should be able to install docker, use the CLI to create a new image, a container, launch an application in it, store data, onfigure the network.
+At the end of the Lab students should be able to install Docker, use the CLI to create a new image, a container, launch an application in it, store data, configure the network.
 
-This Lab is intended to be trial and error so that during the session students should understand really what is behind the tool, instead of blindly following instructions, which never teach people anything IMHO. You've been warned ;-)
+This Lab is intended to be trial and error so that during the session students should understand really what is behind the tool.  Blindly following instructions is not an effective way to learn IMHO. You've been warned ;-)
 
 Expected duration : 120 minutes
 
 ## Reference documents
-When dealing with the installation and configuration of Docker, the first approach is to look at the reference Web site http://docker.io/: 
+When dealing with the installation and configuration of Docker, the first step  is to check the reference Web site http://Docker.io/:
 
```

Comment:
Why not, but not for the Docker vs docker modification, this time, a lower case one is appropriate.